### DISCUSSION TITLE
Auto labeling

### DISCUSSION THREAD

**Discussion by ghost** [spacedriveapp/spacedrive#106] [Feature Request]
It would be really nice if the files were labeled automatically. I have many files in my hard drives and it would be easier to find them if they were labeled. An opt-in telemetry setting for sharing labels is what comes to mind as an option to accomplish this. The labels would all be shared in the same cloud and then every file already labeled in the cloud would be automatically labeled for every user without the need of each user to label those same files themselves.

**Comment by jamiepine** [comment]
this is the plan for "labels", they're like tags but get assigned by Spacedrive automatically. this is only planned in the current database structure though, it's not engineered
